src/core/common/Topbar/fragments/TopPart.js

```diff
@@ -0,0 +1,78 @@
+import React, { Component } from "react";
+
+import { withStyles } from '@material-ui/core/styles';
+import AccountCircle from '@material-ui/icons/AccountCircle';
+import CardMedia from "@material-ui/core/CardMedia";
+import BackIcon from "@material-ui/icons/KeyboardBackspace";
+import Toolbar from '@material-ui/core/Toolbar';
+import IconButton from '@material-ui/core/IconButton';
+import Menu from '@material-ui/core/Menu';
+
+import logo from "../../../images/pulsetile-core-logo.png";
+import styles from "../../../styles";
+
+/**
+ * This component returns white (top) part of Showcase TopBar
+ *
+ * @author Bogdan Shcherban <bsc@piogroup.net>
+ * @constructor
+ */
+class WhitePart extends Component {
```

Comment:
Same, "WhitePart" doesn't convey meaning for someone who doesn't know the look and feel of the app. What does this component do?

Reply:
It was fixed: 
/src/core/common/Topbar/fragments/TopPart.js